apps/showcase/src/app/app.module.ts

```diff
@@ -40,12 +41,15 @@ const runtimeChecks: Partial<RuntimeChecks> = {
 registerLocaleData(localeEN, 'en-GB');
 registerLocaleData(localeFR, 'fr-FR');
 
+const logger = new ConsoleLogger();
+
 function petApiFactory() {
   const apiConfig: ApiClient = new ApiFetchClient(
     {
       basePath: 'https://petstore3.swagger.io/api/v3',
-      requestPlugins: [],
-      fetchPlugins: []
+      requestPlugins: [new SessionIdRequest('Ama-Client-Ref', true, logger)],
```

Comment:
Maybe we could move that to a documentation section 
as it's not needed for this API